win32ss/gdi/ntgdi/path.c

```diff
@@ -1459,7 +1459,7 @@ PATH_PathToRegion(
 
     if (!pPath->numEntriesUsed) return FALSE;
 
-    counts = ExAllocatePoolWithTag(PagedPool, (pPath->numEntriesUsed / 2) * sizeof(counts), TAG_PATH);
+    counts = ExAllocatePoolWithTag(PagedPool, (pPath->numEntriesUsed / 2) * sizeof(*counts), TAG_PATH);
```

Comment:
Especially this one :smile:

Reply:
While here, could you s/*pHrgn/Rgn/ on line 1447?
That is an addendum to 215e0d90a14fd20fea1824293c962100956399d3 (r64177).

--

> Fix a bug

That sounds somewhat worse than it is: "simply" allocating the double of the needed size.
(Unless needed size is enormous, of course.)

Reply:
wait.. is this using an unintialized variable to initialize itself?

Reply:
it's using a **sizeof**, so that thing is well defined at compile time.